sdk/search/search-documents/test/internal/perf/serialization.spec.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import fs from "fs/promises";
+import { Suite } from "mocha";
+import { deserialize, serialize } from "../../../src/serialization";
+
+describe("serialization", async function (this: Suite) {
+  let body: any;
+
+  beforeEach(async () => {
+    body = await fs.readFile("test/internal/perf/a.json", "utf-8").then(JSON.parse);
+  });
+
+  it("should deserialize payloads quickly", () => {
+    const time = new Date();
+    deserialize(body);
+    assert.isAtMost(new Date().getMilliseconds() - time.getMilliseconds(), 200);
+  });
+
+  it("should serialize payloads quickly", () => {
```

Comment:
@HarshaNalluru would this kind of thing be a good candidate for using the perf framework? Not to block this PR but perhaps we could focus on correctness in the unit tests and track the speed of the serialization in a separate perf run

Reply:
Noted, will dig a little deeper